File: ALGORITMO E PROGRAMAÇÃO/LISTA 06/Exe03.js
const entrada = require('prompt-sync')({
    sigint: true
})

function principal(){
    const tamanho_vetor = Number(entrada('Informe o tamanho do vetor: '))
    let vetor_a = Array(tamanho_vetor)
    let vetor_b = Array(tamanho_vetor)

   for(let i = 0; i < tamanho_vetor; i++){
        vetor_a[i] = Number(entrada(`Informe o elemento ${i + 1} do vetor A: `))
    }
   for(let i = 0; i < tamanho_vetor; i++){
        vetor_b[i] = Number(entrada(`Informe o elemento ${i + 1} do vetor B: `))
    }
    const vetor_c = [...vetor_a,...vetor_b]
    
    console.log('Vetor A',vetor_a)
    console.log('Vetor B',vetor_b)
    console.log('Vetor C',vetor_c)
    
}
principal()

File: ESTRUTURA DE DADOS/LISTA03/Exe03.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main() {
  
   int tamanhoA, tamanhoB, i, j,n;
    printf("Informe o tamanho do vetorA: ");
    scanf("%d", &tamanhoA);
    printf("Informe o tamanho do vetorB: ");
    scanf("%d", &tamanhoB);
    int VetorA[tamanhoA], VetorB[tamanhoB];
    int tamanhoC = tamanhoA + tamanhoB;
    int Uniao[tamanhoC];
    for(i = 0; i < tamanhoA; i++) {
        printf("Digite o elemento %d: do vetorA: ",i);
        scanf("%d", &VetorA[i]);
         Uniao[i] = VetorA[i];
    }
    for(j = 0; j < tamanhoB; j++) {
        printf("Digite o elemento %d do vetor B: ",i);
        scanf("%d", &VetorB[j]);
         Uniao[i] = VetorB[j];
          i++;
    }
    printf("União é: {");

      for(n = 0; n < tamanhoC; n++) {
          for(int m = tamanhoC; m >= 0; m--){
            if(Uniao[m] != Uniao[n]){
              printf(" %d ",Uniao[n]);
              break;
            };
          };

      };
    printf("}\n");
     int interseccao[0];
    printf("Interseccao [");
    for(i = 0; i < tamanhoA; i++){
        for(j = 0; j < tamanhoB ; j++){
            if(VetorA[i] == VetorB[j]){
                printf(" %d ", VetorA[i]);                             
             };
        };
    };
    printf("]\n");
    return 0;
}
